pyscript.core/src/plugins/pyterminal.js

```diff
@@ -0,0 +1,110 @@
+// PyScript py-terminal plugin
```

Comment:
shouldn't this be called `py-terminal.js` instead? :thinking:

Reply:
good point 👍